prog/dftb+/prg_dftb/parser.F90

```diff
@@ -2939,6 +2976,9 @@ subroutine readDispDFTD3(node, input)
         & modifier=buffer, child=child)
     call convertByMul(char(buffer), lengthUnits, child, input%cutoffCN)
     call getChildValue(node, "threebody", input%threebody, default=.false.)
+    ! D3H5 - additional H-H repulsion
+    call getChildValue(node, "hhrepulsion", input%hhrepulsion, default=.false.)
```

Comment:
Does it make sense to set this without the H5 correction being also enabled?

Reply:
Yes, it's a modification of the dispersion that can be used without H5, when correction of H-bonds is not needed or a different one is used.